tests/test_edgeql_ddl.py

```diff
@@ -7928,17 +7928,17 @@ async def test_edgeql_ddl_global_type_changes_06(self):
                 'create global foo := [(f := [1])]',
                 [
                     {
-                        'name': 'array<tuple<f:array<std||int64>>>',
+                        'name': 'array<tuple<f:array<std::int64>>>',
```

Comment:
It'd be nice if we inserted a space after the `:` in named tuples, but not critical.